src/App.js

```diff
@@ -1,8 +1,16 @@
-import './App.css';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Detail from './components/Detail';
 
 function App() {
   return (
-    <h1>NAS</h1>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/detail/:userId" element={<Detail />} />
+      </Routes>
```

Comment:
The project contains a Home page and a Detail page 🤓 Well done!

Reply:
Thank you :)